public/china/index.html

```diff
@@ -4,41 +4,64 @@
   <title>China</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <!-- <link rel="stylesheet" href="../stylesheets/style.css"> -->
+  <link rel="stylesheet" href="../stylesheets/style.css">
 </head>
 <body>
   <header id="header">
-    <nav id="navigation">
-      <ul>
-        <li><a href="../">Home</a></li>
-        <li><a href="../china/">China</a></li>
-        <li><a href="../japan/">Japan</a></li>
-        <li><a href="../malaysia/">Malaysia</a></li>
-        <li><a href="../singapore/">Singapore</a></li>
-        <li><a href="../south_korea/">South Korea</a></li>
-        <li><a href="../thailand/">Thailand</a></li>
-      </ul>
+    <!-- Sidebar -->
+    <nav class="sidebar barmenu topheader" id="mySidebar">
+      <a onclick="close_menu()" class="button">Close Menu</a>
```

Comment:
"written leveraging event handlers and wholly avoiding deprecated JavaScript attributes in your HTML, such as onload" -- please make sure onclick is not deprecated.